### PR TITLE
Zero displayed as blank

### LINKED ISSUE #16: Zero is displayed as blank cell
in `PoiCellDefinition`, there is:
```
   @Override
    void value(Object value) {
        if (!value) {
            xssfCell.setCellType(Cell.CELL_TYPE_BLANK)
            return
        }
        ....
    }
```

Looks like `0` falls into the first check.
I think, the correct way is to compare `value` to `null` explicitly.

### PR DESCRIPTION
**Pull request by atapin**
Fixes #16

### CODE REVIEW

File: spreadsheet-builder-poi/src/main/groovy/org/modelcatalogue/spreadsheet/builder/poi/PoiCellDefinition.groovy [hunk 1]
```diff
@@ -130,12 +130,6 @@ class PoiCellDefinition extends AbstractCellDefinition implements Resolvable, Sp
             return
         }
 
-        if (value instanceof Boolean) {
```
Comment (atapin) [inline spreadsheet-builder-poi/src/main/groovy/org/modelcatalogue/spreadsheet/builder/poi/PoiCellDefinition.groovy]: @musketyr there's exactly the same one right before

Comment (musketyr) [inline spreadsheet-builder-poi/src/main/groovy/org/modelcatalogue/spreadsheet/builder/poi/PoiCellDefinition.groovy]: right, I can see it in the full file. not sure why did GitHub mark it as red

**Comment by musketyr** [comment]
👍

File: spreadsheet-builder-poi/src/main/groovy/org/modelcatalogue/spreadsheet/builder/poi/PoiCellDefinition.groovy (hunk 1 above)
Comment (musketyr) [inline spreadsheet-builder-poi/src/main/groovy/org/modelcatalogue/spreadsheet/builder/poi/PoiCellDefinition.groovy]: now it make sense :-) thanks again

**Pull request merged**